source/operators/rekognition/check_custom_label_detection_status.py

```diff
@@ -0,0 +1,129 @@
+# Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+###############################################################################
+# PURPOSE:
+#   Lambda function to check the status of a Rekognition job and save that job's
+#   data to the MIE dataplane when the job is complete.
+###############################################################################
+
+import os
+import boto3
+import json
+from botocore import config
+from MediaInsightsEngineLambdaHelper import OutputHelper
+from MediaInsightsEngineLambdaHelper import MasExecutionError
+from MediaInsightsEngineLambdaHelper import DataPlane
+
+operator_name = os.environ['OPERATOR_NAME']
+output_object = OutputHelper(operator_name)
+
+mie_config = json.loads(os.environ['botoConfig'])
+config = config.Config(**mie_config)
+rek = boto3.client('rekognition', config=config)
+
+def lambda_handler(event, context):
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

This function contains 71 lines of code, not including blank lines or lines with only comments, Python punctuation characters, identifiers, or literals. By comparison, 99% of the functions in the CodeGuru reference dataset contain fewer lines of code. Large functions might be difficult to read and have logic that is hard to understand and test. We recommend that you simplify this function or break it into multiple functions.